.github/pr_issue_template.md

```diff
@@ -0,0 +1,63 @@
+## Related Issue 
+  - Info about the related issue 
+
+Closes: #[issue number that will be closed through this PR]
+
+#### Describe the changes you've made
+
+Give a clear description what modifications you have made
+
+## Type of change
+
+What sort of change have you made :
+<!--
+Example how to mark a checkbox :-
+- [x] My code follows the code style of this project.
+-->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Code style update (formatting, local variables)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Describe how have you verified the changes made
+
+## Checklist:
+<!--
+Example how to mark a checkbox :-
+- [x] My code follows the code style of this project.
+-->
+- [ ] My code follows the guidelines of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly whereever it was hard to understand.
+- [ ] I have made corresponding changes to the documentation.
+- [ ] My changes generate no new warnings.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
+- [ ] Any dependent changes have been merged and published in downstream modules.
+
+## Screenshots
+
+ Original           | Updated
+ :--------------------: |:--------------------:
+ **original screenshot**  | <b>updated screenshot </b> |
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+
+
+
+
+
+
+
+
+
+
```

Comment:
Why so many empty lines?

Reply:
I don't know I actually checked for it before making a pr dont know why they are appearing will be checking soon sir

Reply:
Could be just a glitch. Don't worry, just remove and send a commit! 😁